### PR TITLE
Docs for Mesh use cases

### PR DESCRIPTION
**Pull request by medakk**
This adds docs for mesh use cases and updates all youtube links to mp4 files. The syntax is a little clunky but I couldn't find a plugin that handled this directly. Its still not too bad:

```html
 <p><video src="https://sci.utah.edu/~shapeworks/doc-resources/mp4s/femur_option1.mp4" autoplay muted loop style="width:100%"></p> 
```

### CODE REVIEW

**Comment by sheryjoe** [comment]
Looks nice!

File: docs/dev/mkdocs.md [hunk 1]
```diff
@@ -16,6 +16,9 @@ See [this link](https://mkdocs.github.io/mkdocs-bootswatch/) for a list of boots
 Next, install font awesome for icons to display in the documentation.   
 `pip install fontawesome-markdown`
```
Comment (cchriste) [inline docs/dev/mkdocs.md:18]: Can we please make it more clear to the reader that these dependencies are already installed by the conda_installs script. Maybe describing the deps is still worth including, but it's confusing as it's worded right now.

File: docs/getting-started/interfaces.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ $ conda activate shapeworks
 ShapeWorksStudio is a cross-platform graphical user interface (GUI) to support the standard shape analysis workflow needed by most ShapeWorks users, enabling a wide variety of research scenarios. It is currently supporting image-based grooming and is actively under development to support surface meshes, more sophisticated grooming operations (including user annotations), and offline processing.  ShapeWorksStudio enables real-time parameter tuning and visualization of the optimization process and statistical analyses.
 
 
-[![ShapeWorksStudio](../img/getting-started/studio_thumbnail.png)](https://youtu.be/RX4uSB_ot3U "ShapeWorksStudio - Click to Watch!")
+<p><video src="https://sci.utah.edu/~shapeworks/doc-resources/mp4s/studio_demo.mp4" autoplay muted loop style="width:100%"></p>
```
Comment (cchriste) [inline docs/getting-started/interfaces.md:33]: These are great! No more YouTube ads when trying to rtfm. :P 
Where are the videos stored at SCI? Is this location noted anyway in the documentation?

**Review by cchriste — COMMENTED** [review]
Just a couple questions in my comments with the files, and one overall question:

File: docs/dev/mkdocs.md (hunk 1 above)
Comment (sheryjoe) [reply docs/dev/mkdocs.md:18]: That is the first line in this section
**Important: The below installations are added to conda_insalls scripts.**

Comment (sheryjoe) [reply docs/dev/mkdocs.md:18]: Just fixed the typo in "installs"

**Comment by sheryjoe** [comment]
> how can I look at the compiled version of these docs? (you might simply direct me to the place where this is explained in the docs)

You can find this link in the main readme.

Please visit [ShapeWorks website](http://sciinstitute.github.io/ShapeWorks/) for more information.

> I can open the individual .md files, but can't really give a final review until I've seen it in action. 

I think the only way you can see it in action is to checkout the branch and run mkdocs serve. These changes won't be reflected in the compiled version (i.e., gh-pages branch) till it is merged to master. @akenmorris ?

**Comment by akenmorris** [comment]
> I think the only way you can see it in action is to checkout the branch and run mkdocs serve. 

I think so.  Unless you have a location to place each individual branch's output.

File: docs/getting-started/interfaces.md (hunk 2 above)
Comment (sheryjoe) [reply docs/getting-started/interfaces.md:33]: Good point. That was on the [internal todo list](https://sciinstitute.github.io/ShapeWorks/todo/) for docs. I submitted an issue now #703.

**Comment by sheryjoe** [comment]
> > I think the only way you can see it in action is to checkout the branch and run mkdocs serve.
> 
> I think so. Unless you have a location to place each individual branch's output.

It is not clear to me how this would be feasible as we only have one gh-pages branch, unless mkdocs build can be directed to build the site to a specific local directory. But, this still could be accomplished by mkdocs serve where the `site` folder should contain your local version of the compiled docs.

**Comment by cchriste** [comment]
> > how can I look at the compiled version of these docs? (you might simply direct me to the place where this is explained in the docs)
> 
> You can find this link in the main readme.
> 
> Please visit [ShapeWorks website](http://sciinstitute.github.io/ShapeWorks/) for more information.
> 
> > I can open the individual .md files, but can't really give a final review until I've seen it in action.
> 
> I think the only way you can see it in action is to checkout the branch and run mkdocs serve. These changes won't be reflected in the compiled version (i.e., gh-pages branch) till it is merged to master. @akenmorris ?

I think it's _completely_ reasonable to check out the branch and run `mkdocs serve` then browse to localhost:8000. 
I recommend explaining this right at the top of the pull request so people don't have to dig through the developer docs to remember the command every time they want to thoroughly review the updates.

File: docs/dev/mkdocs.md (hunk 1 above)
Comment (cchriste) [reply docs/dev/mkdocs.md:18]: I just slightly modified this to clarify the packages have already been installed.

**Review by cchriste — APPROVED** [review]
Looks great!

**Comment by cchriste** [comment]
Can this be merged now?

**Pull request merged**